lib/msf/core/payload/windows/meterpreter_loader.rb

```diff
@@ -28,6 +28,11 @@ def initialize(info = {})
       'PayloadCompat' => { 'Convention' => 'sockedi handleedi -https', },
       'Stage'         => { 'Payload'   => "" }
       ))
+    register_options(
+      [
+        OptBool.new('DEBUG', [false, 'Use the Meterpreter debug build.'])
```

Comment:
`Debug` may be too generic, potentially `MeterpreterDebug`, or `MeterpreterDebugBuild`? 🤔 

```
msf6 payload(osx/x64/meterpreter_reverse_http) > set Meterpreter
set MeterpreterPrompt     set MeterpreterTryToFork 
```

Python has its own:

```
 set PythonMeterpreterDebug 
```

But potentially these could be aliases?